db.go

```diff
@@ -52,6 +52,27 @@ var migrations = &migrate.MemoryMigrationSource{
 				`,
 			},
 		},
+		{
+			Id: "3",
+			Up: []string{
+				`CREATE TABLE IF NOT EXISTS forwarding_history (
+                                        add_time TIMESTAMP NOT NULL,
+                                        resolved_time TIMESTAMP NOT NULL,
+                                        settled BOOLEAN NOT NULL,
+                                        incoming_amt_msat INTEGER NOT NULL CHECK (incoming_amt_msat >= 0),
+                                        outgoing_amt_msat INTEGER NOT NULL CHECK (outgoing_amt_msat >= 0),
+                                        incoming_peer TEXT NOT NULL,
+                                        incoming_channel INTEGER NOT NULL,
+                                        incoming_htlc_index INTEGER NOT NULL,
+                                        outgoing_peer TEXT NOT NULL,
+                                        outgoing_channel INTEGER NOT NULL,
+                                        outgoing_htlc_index INTEGER NOT NULL,
+                                        
+                                        CONSTRAINT unique_htlc_combination UNIQUE (incoming_channel, incoming_htlc_index, outgoing_channel, outgoing_htlc_index)
```

Comment:
Hm, I wasn't really worried about the fundamental issue of having duplicate circuit keys (since that means something is very wrong in LND), but I guess we could end up with a bug in circuitbreaker that pairs them up incorrectly? Will split up into two constraints.